src/adhocracy_mercator/adhocracy_mercator/resources/root.py

```diff
@@ -19,6 +21,21 @@ def add_mercator_process(context: IPool, registry: Registry, options: dict):
                             appstructs=appstructs)
 
 
+def initialize_workflow(context: IPool, registry: Registry, options: dict):
+    """Initialize mercator workflow."""
+    root = context
+    root.__acl__ = [(Allow, 'role:god', ALL_PERMISSIONS)]
+    request = Request.blank('/dummy')
+    request.root = root
+    request.registry = registry
+    request.__cached_principals__ = ['role:god']
+    mercator_process = root['mercator']
+    workflow = registry.content.workflows['mercator']
+    workflow.initialize(mercator_process)
+    workflow.transition_to_state(mercator_process, request, 'announce')
+    workflow.transition_to_state(mercator_process, request, 'participate')
```

Comment:
The proper way would be to change the inital state for the mercator workflow. I propose we: 
* move this code to a helper function (we will need this again)
* add field 'state_initial' to the workflow schema  to set the inital state instead of "missusing"  the state_order field.